sections/footer.liquid

```diff
@@ -77,15 +77,15 @@
                   <div class="footer-block__details-content footer-block-image {{ block.settings.alignment }}">
                     {%- if block.settings.image != blank -%}
                       {%- assign image_size = block.settings.image_width -%}
-                      {%- assign image_size_2x = block.settings.image_width | times: 2 -%}
+                      {%- assign image_size_2x = image_size | times: 2 | at_most: 5760 -%}
```

Comment:
`at_most: 5760` was added to the footer and header to avoid width errors:

- Originally mentioned here: https://github.com/Shopify/dawn/pull/955#discussion_r759489837

<details>
  <summary>Screenshot of width error</summary>

  ![image](https://user-images.githubusercontent.com/28404165/155213958-30871aa5-8f8c-4879-9b35-b8159ad66eb6.png)

</details>